tests/__init__.py

```diff
@@ -6,9 +6,11 @@
 try:
     import numpy
     numpy_version = numpy.version.version
+    integer_types = [int, numpy.int16, numpy.int32, numpy.int64]
 except ImportError:
     numpy = False
     numpy_version = 'not available'
+    integer_types = None
```

Comment:
Originally the idea was that `integer_types` was only used in numpy tests. I just refactored this pull request differently though.